xata/package.json

```
@@ -0,0 +1,31 @@
+{
+  "private": true,
+  "sideEffects": false,
+  "scripts": {
+    "build": "remix build",
+    "dev": "remix dev",
+    "start": "remix-serve build",
+    "xata:start": "xata init --schema=schema.template.json",
+    "xata:codegen": "xata codegen"
+  },
+  "dependencies": {
+    "@remix-run/node": "latest",
+    "@remix-run/react": "latest",
+    "@remix-run/serve": "latest",
+    "@xata.io/client": "latest",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
```

Let's keep this as we have in `_template`

```suggestion
    "@remix-run/node": "*",
    "@remix-run/react": "*",
    "@remix-run/serve": "*",
    "react": "^17.0.2",
    "react-dom": "^17.0.2"
```